From cb7a8033014300f3c538089d1ef3b9b8dd868739 Mon Sep 17 00:00:00 2001
From: Leonard Crestez <leonard.crestez@nxp.com>
Date: Fri, 9 Nov 2018 12:55:25 +0200
Subject: [PATCH 5080/5242] MLK-20309 ARM64: dts: Remove
 fsl-imx8mm-evk-inmate.dtb from makefile

commit  12bfac0d4969821e73bbcac7b0082d2e860b1cc8 from
https://source.codeaurora.org/external/imx/linux-imx.git

The dts file was removed but makefile was not updated so build broke.
Fix by removing from makefile.

Fixes: a9c2aa010df8 ("MLK-20252 ARM64: dts: correct imx8mm root memory")

Signed-off-by: Leonard Crestez <leonard.crestez@nxp.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/Makefile |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/Makefile b/arch/arm64/boot/dts/freescale/Makefile
index 61ff5f1..0dcdb5b 100644
--- a/arch/arm64/boot/dts/freescale/Makefile
+++ b/arch/arm64/boot/dts/freescale/Makefile
@@ -123,7 +123,6 @@ dtb-$(CONFIG_ARCH_FSL_IMX8MM) += fsl-imx8mm-evk.dtb \
 				 fsl-imx8mm-ddr4-val.dtb \
 				 fsl-imx8mm-evk-rm67191.dtb \
 				 fsl-imx8mm-evk-root.dtb \
-				 fsl-imx8mm-evk-inmate.dtb \
 				 fsl-imx8mm-evk-revb.dtb \
 				 fsl-imx8mm-evk-revb-rm67191.dtb \
 				 fsl-imx8mm-ddr4-evk-rm67191.dtb
-- 
1.7.9.5

